### DISCUSSION TITLE
Aggregation

### DISCUSSION THREAD

**Discussion by bhuvnkumar** [prisma/prisma#18093] [Q&A] [answered]
## Problem
Can you support aggregation like postgres raw query.
select sum(col1*col2) from table;
It can be done as with support inner arithmetic operations.

## Suggested solution

const aggregations = await prisma.user.aggregate({
  _sum: {
    weightage:{_multiply:[col1,col2]}
  },
})

**Comment by nurul3101** [comment, marked as answer]
Hey @Bhuvnquo 👋 

Prisma support [atomic operations](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#atomic-number-operations) which you can use to multiply a column value with a static value. Looking at your use case it looks like you would need to use [Raw Query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access).

**Reply by bhuvnkumar** [reply]
Yes, @nurul3101  Thanks for answering.
I am able to do using raw query but I am supposing it should be in Prisma itself.

**Reply by nurul3101** [reply]
I don't think it's natively possible yet, the raw query would be the only workaround for now.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
